### PR TITLE
Avoid structures to spawn in ocean biomes

### PR DESCRIPTION
**Pull request by jimmikaelkael**
Fix for #18. 
I have to admit that's me the author of the culprit line :)

### CODE REVIEW

File: src/main/java/net/glowstone/generator/structures/GlowTemple.java [hunk 1]
```diff
@@ -26,8 +26,6 @@ public GlowTemple(World world, int chunkX, int chunkZ) {
     public GlowTemple(World world, Random random, int chunkX, int chunkZ) {
         this(world, chunkX, chunkZ);
 
-        types.put(Biome.OCEAN, TempleType.values()[random.nextInt(3)]); // fix for lack of biomes
```
Comment (mastercoms) [inline src/main/java/net/glowstone/generator/structures/GlowTemple.java]: Sorry, you're the expert here, but would you not need to do a new types.put statement to specify the biomes temples can spawn in?

Comment (jimmikaelkael) [inline src/main/java/net/glowstone/generator/structures/GlowTemple.java]: Already done is class static. At the time I made this, there was only the ocean biome on glowtone, it explains the line artifact, but the types list is already fed ;)

Comment (mastercoms) [inline src/main/java/net/glowstone/generator/structures/GlowTemple.java]: Yes, I remember those days. Good times.

**Comment by jimmikaelkael** [comment]
Unfortunately, the fix hasn't been committed: https://github.com/deathcap/GlowstonePlusPlus/commits/master/src/main/java/net/glowstone/generator/structures/GlowTemple.java

**Comment by deathcap** [comment]
huh, https://github.com/deathcap/GlowstonePlusPlus/commit/74f0e1f8cbc752c88c5184203e7c08f20b967b87 is detached from master. @mastercoms ?

Re-merged in bb01c49008e19dd5d95f4d8dd50b053fe2565b2c

**Comment by mastercoms** [comment]
Yeah, idk. Thank for merging it.

**Pull request merged**